src/adapter/db/user.mapper.ts

```diff
@@ -1,34 +1,26 @@
-import { User, UserCredential, UserListItem } from 'src/domain/user';
+import { User, UserRole } from 'src/domain/user';
 
 import { UserDbEntity } from './user.entity';
 
 export const mapToUser = (dbEntity: UserDbEntity): User => {
   return {
     id: dbEntity.id,
+    snsId: dbEntity.snsId,
     email: dbEntity.email,
     role: dbEntity.role,
-    firstName: dbEntity.firstName,
-    lastName: dbEntity.lastName,
+    nickname: dbEntity.nickname,
     profileImageUrl: dbEntity.profileImageUrl,
     createdAt: dbEntity.createdAt,
   };
 };
 
-export const mapToUserCredential = (dbEntity: UserDbEntity): UserCredential => {
-  return {
-    id: dbEntity.id,
-    email: dbEntity.email,
-    password: dbEntity.password,
-  };
-};
-
-export const mapToUserListItem = (dbEntity: UserDbEntity): UserListItem => {
-  return {
-    id: dbEntity.id,
-    email: dbEntity.email,
-    role: dbEntity.role,
-    firstName: dbEntity.firstName,
-    lastName: dbEntity.lastName,
-    createdAt: dbEntity.createdAt,
-  };
+export const mapToUserDbEntity = (user: User): UserDbEntity => {
+  const entity = new UserDbEntity();
+  entity.snsId = user.snsId;
+  entity.nickname = user.nickname;
+  entity.profileImageUrl = user.profileImageUrl;
+  entity.email = user.email;
+  entity.role = user.role ?? UserRole.USER;
+  entity.createdAt = new Date();
```

Comment:
회원가입시 기본값이 필요하다면 gateway의 메소드에 작성해주세요!

```suggestion
  entity.role = user.role;
```